src/tools/snapshot/SnapshotBuffer.java

```diff
@@ -1,47 +1,48 @@
 package tools.snapshot;
 
-import org.graalvm.collections.EconomicMap;
-
+import som.interpreter.actors.Actor.ActorProcessingThread;
 import som.interpreter.objectstorage.ClassFactory;
+import som.vm.VmSettings;
 import som.vm.constants.Classes;
 import tools.concurrency.TraceBuffer;
+import tools.concurrency.TracingActors.TracingActor;
 import tools.replay.nodes.TraceActorContextNode;
 import tools.snapshot.deserialization.DeserializationBuffer;
 
 
 public class SnapshotBuffer extends TraceBuffer {
 
-  public static final int FIELD_SIZE    = 8;
-  public static final int CLASS_ID_SIZE = 2;
-  public static final int MAX_FIELD_CNT = Byte.MAX_VALUE;
-
-  /**
-   * This map allows us to know if we already serialized an object (and avoid circles).
-   * We can get the location of the serialized object in the trace
-   */
-  private final EconomicMap<Object, Long> entries;
+  public static final int               FIELD_SIZE    = 8;
+  public static final int               CLASS_ID_SIZE = 2;
+  public static final int               MAX_FIELD_CNT = Byte.MAX_VALUE;
```

Comment:
Grouping of fields is used for formatting, so, avoiding to group static and non-static fields helps with avoiding unnecessary whitespace changes.

It also emphasizes that there is a difference and makes the different types of things stand out.